plugins/woocommerce/src/Internal/Admin/Onboarding/OnboardingSetupWizard.php

```diff
@@ -230,6 +230,9 @@ public function component_settings( $settings ) {
 		$settings['onboarding']['euCountries']     = WC()->countries->get_european_union_countries();
 		$settings['onboarding']['localeInfo']      = include WC()->plugin_path() . '/i18n/locale-info.php';
 		$settings['onboarding']['profile']         = $profile;
+		$settings['onboarding']['pageCount']       = (int) ( wp_count_posts( 'page' ) )->publish;
```

Comment:
nitpick: Should we include these values only if users are in the setup wizard by checking '$this->is_setup_wizard()' ?

Reply:
@moon0326  Good idea, I think it's already checked a few lines above? 

https://github.com/woocommerce/woocommerce/blob/483ae18bcac6f32008e03ce9f68666e0562996db/plugins/woocommerce/src/Internal/Admin/Onboarding/OnboardingSetupWizard.php#L216-L223

Reply:
@adrianduffell It does, but it continues if there is a visible task. I thought we could limit the count calls to the OBW. There can be a case where a big store still has tasks left to do. I think this is going to be okay as count calls are cached. I was just being extra cautious as we had a similar issue before.

Reply:
Ah yeah I see what you mean. We could wrap the new properties in `if ( this->is_setup_wizard() ) {}` as a quick follow-up